package.json

```diff
@@ -1,28 +1,33 @@
 {
-    "name": "@pxblue/types",
+    "name": "@brightlayer-ui/types",
     "version": "2.0.0",
-    "description": "Type definitions shared between PX Blue packages",
+    "description": "Type definitions shared between Brightlayer UI packages",
     "main": "dist/index.js",
     "types": "dist/index.d.ts",
     "scripts": {
         "test": "./buildTest.sh",
-        "build": "rm -rf dist && tsc"
+        "build": "rm -rf dist && tsc",
+        "publish:package": "set npm_config_yes=true && npx -p @brightlayer-ui/publish blui-publish",
+        "tag:package": "npx -p @brightlayer-ui/tag blui-tag"
     },
     "repository": {
         "type": "git",
-        "url": "git+https://github.com/pxblue/types.git"
+        "url": "git+https://github.com/brightlayer-ui/types.git"
     },
     "keywords": [
-        "PXBlue",
+        "BLUI",
         "types"
```

Comment:
Curious if we want to leave PX Blue in the keywords for people to be able to find packages if they haven't heard about the update? I'm fine either way.

Reply:
Huayun brought up the same point but all PXB packages will be hit with a deprecation notification, pointing users to the new packages. I don't think it's required.